src/server/services/wfs/qgswfsutils.cpp

```diff
@@ -100,15 +100,16 @@ namespace QgsWfs
     return nullptr;
   }
 
-  QgsFeatureRequest parseFilterElement( const QString &typeName, QDomElement &filterElem, const QgsProject *project )
+  QgsFeatureRequest parseFilterElement( const QString &typeName, QDomElement &filterElem, const QgsProject *project, QStringList *serverFids )
```

Comment:
Why using a pointer instead of a reference for `serverFids`? Because in this case, you should check that it's not a null pointer before calling it.

Reply:
It's not gonna be null because of the default value (which will be leaked if used)

Reply:
Rho yes, I didn't see that :)

Reply:
Pointer because project has a default value and serverFids also.

Reply:
I have updated the method to use const QStringList for serverFids.

Reviews are welcome.